gcp-observability/src/test/java/io/grpc/gcp/observability/logging/GcpLogSinkTest.java

```diff
@@ -97,34 +96,29 @@ public class GcpLogSinkTest {
           String.valueOf(EventLogger.LOGGER_CLIENT)).build())
       .putFields("rpc_id", Value.newBuilder().setStringValue(rpcId).build())
       .build();
+  @Mock
   private Logging mockLogging;
 
-  @Before
-  public void setUp() {
-    mockLogging = mock(Logging.class);
-  }
-
   @Test
   public void createSink() {
-    Sink mockSink = new GcpLogSink(mockLogging, destProjectName, locationTags,
-        customTags, flushLimit);
-    assertThat(mockSink).isInstanceOf(GcpLogSink.class);
+    GcpLogSink sink = new GcpLogSink(mockLogging, destProjectName, locationTags,
+        customTags, FLUSH_LIMIT);
+    assertThat(sink).isInstanceOf(Sink.class);
```

Comment:
What's the value of this assert? Since `GcpLogSink` is declared to implement `Sink` this will be true so the test seems trivial - does not verify any run time behavior

Reply:
Makes sense. Removed the test since this was not adding any additional value.